Integration/Iterator/ProductIterator.php

```diff
@@ -6,6 +6,7 @@
 use Akeneo\PimEnterprise\ApiClient\AkeneoPimEnterpriseClientInterface;
 use Gaufrette\Filesystem;
 use Psr\Log\LoggerInterface;
+use Akeneo\Pim\ApiClient\Exception\NotFoundHttpException;
```

Comment:
Make sure to order import statements please

Reply:
done